uint8 NONE = 0
uint8 RISK = 1
uint8 NO_RISK = 2

uint8 NO_ACTION = 0
uint8 REQUEST = 1

unique_identifier_msgs/UUID object_id
uint8 intervention
uint8 action
float64 distance
uint8 path_index
